src/main/java/de/zalando/aruha/nakadi/partitioning/HashPartitionStrategy.java

```diff
@@ -28,6 +29,7 @@ public String calculatePartition(final EventType eventType, final JSONObject eve
             final int hashValue = partitionKeyFields.stream()
                     // The problem is that JSONObject doesn't override hashCode(). Therefore convert it to
                     // a string first and then use hashCode()
+                    .map(pkf -> EventCategory.DATA.equals(eventType.getCategory()) ? "data." + pkf : pkf)
```

Comment:
Maybe we need to create constant field with value `data` just to know were exactly it's used. 
I've found only EventTypeValidation#wrapSchemaInData, and it will be hard for me to understand where else it could happen.